qa/integration/services/dockerized/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:stretch
+##
+# Define a base image for all service images.
+##
+
+ENV _JAVA_OPTIONS "-Djava.net.preferIPv4Stack=true"
+ENV TERM=linux
+
+RUN apt-get update && apt-get install -y curl openjdk-8-jre-headless netcat
+
+RUN adduser --disabled-password --gecos '' tester
```

Comment:
It turns out that ```es.insecure.allow.root``` property no longer works in ES 5.x. I found people commenting about that but I could not see the information on ES manual.

Indeed, it does not work:
```
root@56a394927ff8:~# ./elasticsearch/bin/elasticsearch --version
Picked up _JAVA_OPTIONS: -Djava.net.preferIPv4Stack=true -Des.insecure.allow.root=true
Version: 5.0.1, Build: 080bb47/2016-11-11T22:08:49.812Z, JVM: 1.8.0_131
root@56a394927ff8:~# ./elasticsearch/bin/elasticsearch                              
Picked up _JAVA_OPTIONS: -Djava.net.preferIPv4Stack=true -Des.insecure.allow.root=true
[2017-05-31T15:07:01,000][WARN ][o.e.b.ElasticsearchUncaughtExceptionHandler] [] uncaught exception in thread [main]
org.elasticsearch.bootstrap.StartupException: java.lang.RuntimeException: can not run elasticsearch as root
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:116) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:103) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.cli.SettingCommand.execute(SettingCommand.java:54) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:96) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.cli.Command.main(Command.java:62) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:80) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:73) ~[elasticsearch-5.0.1.jar:5.0.1]
Caused by: java.lang.RuntimeException: can not run elasticsearch as root
	at org.elasticsearch.bootstrap.Bootstrap.initializeNatives(Bootstrap.java:96) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:155) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:286) ~[elasticsearch-5.0.1.jar:5.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:112) ~[elasticsearch-5.0.1.jar:5.0.1]
	... 6 more
```

Unless I am missing something here, I can see 3 solutions:
1. Stick with first implementation and services that need ```root``` can use `su` in their scripts.
2. Do the other way around: the docker base image comes with ```root``` only, the services that requires non-root will be in charge of creating an user.
3. A variation of case 2: the docker base image setup a regular user along with ```root``` (which defaults), the services requiring non-root simply switch to the regular user.

Notice that either creating or switching to a non-root need to be carried out in scripts since the ```ONBUILD``` commands are the first executed.